default_warehouse_from_sale_team/__openerp__.py

```diff
@@ -15,6 +15,7 @@
         'sales_team',
         'base_action_rule',
         'purchase_requisition',
+        'warning',
```

Comment:
@ysantiago 
this tests was tested only installing this module?

I think we do not need to depend to that module, maybe you can test using another user demo